web/skins/classic/views/report_event_audit.php

```diff
@@ -102,12 +102,12 @@
     if ( $gap > $EventsByMonitor[$event['MonitorId']]['MaxGap'] )
       $EventsByMonitor[$event['MonitorId']]['MaxGap'] = $gap;
 
-  } # end if has previous events
-  if ( ! $Event->file_exists() ) {
-    $EventsByMonitor[$event['MonitorId']]['FileMissing'][] = $Event;
-  } else if ( ! $Event->file_size() ) {
-    $EventsByMonitor[$event['MonitorId']]['ZeroSize'][] = $Event;
-  }
+  } # end if has previous events # DISABLED takes very long time on busy disk sotarage when saving images
```

Comment:
Agree, and there is a help tip in the zmaudit options

> This is recommended for most systems however if you have a very large number of events the process of scanning the database and filesystem may take a long time and impact performance. In this case you may prefer to not have zmaudit running unconditionally and schedule occasional checks at other, more convenient, times. 

I need to disable zmaudit but not "missing files check" in source, I'm feeling stupid now

> and is the actual reason this view exists.

Well, If I disable audit checks then audit report page shoud disappear, right?

Reply:
OK I got it, you can run zmaudit manually and use report page to view results, but if zmaudit doing same thing about missing files, than why is nessesary that filtering events checking files additionally? It can be stored in database?